phpstan.neon

```diff
@@ -0,0 +1,8 @@
+parameters:
+	level: 0
+	paths:
+		- app
```

Comment:
Ah I forgot this in the cdbxml service. I'll try to add it later today.
I also just put the config directly in the phing task but I see the advantage of doing it this way 👍